flipperdevices/flipperzero-firmware
# [FL-3471] Infrared: buttons move feature rework

# What's new

- Reworked buttons reordering to match similar feature in archive favorites

# Verification 

- Open infrared remote with multiple buttons
- Check edit -> move button option is working

# Checklist (For Reviewer)
